db/migration/1726588731621-MigrateOutdatedConfigsToLatestVersion.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class MigrateOutdatedConfigsToLatestVersion1726588731621
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        // we have v3 configs in the database; turn these into v5 configs
+        // by removing the `data` and `hideLinesOutsideTolerance` properties
+        await queryRunner.query(
```

Comment:
Would it be very annoying to just run our migration code here? It seems that this is a bit fragile and while it will work on prod if we check first, if we just use our migration code then this could work also on old, outdated staging servers or similar if we ever wanted to use it in such a context.

Reply:
yes, we can also just run the migration code here 👍🏻